index.js

```diff
@@ -1790,6 +1790,30 @@
   S.unlines =
   def('unlines', [List], R.compose(R.join(''), R.map(R.concat(_, '\n'))));
 
+
+  //# unfoldr :: (b -> Maybe (a, b)) -> b -> [a]
+  //.
+  //. The unfoldr function is a `dual' to foldr: while foldr reduces a list to
+  //. a summary value, unfoldr builds a list from a seed value. The function
+  //. takes the element and returns Nothing if it is done producing the list or
+  //. returns Just (a,b), in which case, a is a prepended to the list and b is
+  //. used as the next element in a recursive call.
```

Comment:
It's strange to mention `foldr`, a name that doesn't appear here or in Ramda. Could you have a go at rewriting this to reference `R.reduce`?